scrapers/ep_votes/scrapers.py

```diff
@@ -137,9 +137,37 @@ def _extract_data(self) -> MemberInfo:
             first_name=first,
             last_name=last,
             date_of_birth=self._date_of_birth(),
+            facebook=self._facebook(),
+            twitter=self._twitter(),
+            email=self._email(),
             country=self._country(),
         )
 
+    def _facebook(self) -> Optional[str]:
+        tag = self._resource.select_one("a.link_fb")
+        if tag is None:
+            return None
+        return tag["href"]
+
+    def _twitter(self) -> Optional[str]:
+        tag = self._resource.select_one("a.link_twitt")
+        if tag is None:
+            return None
+        return tag["href"]
+
+    def _email(self) -> Optional[str]:
+        tags = self._resource.select("a.link_email")
+        if tags is None:
+            return None
+        addresses = [tag["href"].replace("[dot]", ".") for tag in tags]
+        addresses = [address.replace("[at]", "@") for address in addresses]
+        addresses = [address.replace("mailto:", "") for address in addresses]
+        addresses = [address[::-1] for address in addresses]
```

Comment:
You could use this selector to get only the link containing the parliament email address: `.link_email[href$="europarl.europa.eu"]`

Then you wouldn’t need to loop and filter.

Reply:
I fear this is not supported. In case you want to try, this is the href of Sonneborn: "mailto:ue[dot]aporue.lraporue[at]nrobennos.nitram"